tests/core/tools.js

```diff
@@ -1111,6 +1111,21 @@
 			} );
 		},
 
+		// (#5158)
+		'test convertToPx works after calculator element was removed': function() {
+			var firstResult = CKEDITOR.tools.convertToPx( '10px' );
+			// Based on convertToPx implementation
+			// calculator is the last element under `body` after `convertToPx` invocation.
+			var bodyChildren = CKEDITOR.document.getBody().getChildren(),
+				calculator = bodyChildren.getItem( bodyChildren.count() - 1 );
+
+			calculator.remove();
+
+			var result = CKEDITOR.tools.convertToPx( '10px' );
+			assert.areSame( firstResult, result, 'convertToPx() returns different values when helper calculator was removed.' );
+			assert.areSame( result, 10, 'calculator was not properly attached to DOM.' );
```

Comment:
The step comparing the first result and the second one is unnecessary. Also, the proper order of assertion method parameters is _expected_, _actual_.